app/src/main/java/io/homeassistant/companion/android/sensors/SensorDetailFragment.kt

```diff
@@ -16,16 +16,18 @@ import io.homeassistant.companion.android.database.sensor.SensorDao
 
 class SensorDetailFragment(
     private val sensorManager: SensorManager,
+    private val basicSensor: SensorManager.BasicSensor,
     private val sensorId: String
 ) :
     PreferenceFragmentCompat() {
 
     companion object {
         fun newInstance(
             sensorManager: SensorManager,
+            basicSensor: SensorManager.BasicSensor,
             sensorId: String
```

Comment:
We should remove this as a parameter and change any uses to `basicSensor.id` instead

Reply:
Ah yup no longer needed now :)